### PR TITLE
Fixing  IDENTITY-4176,4177

### PR DESCRIPTION
**Pull request by ChamaraPhilipsuom**
Fixed veracode issues according to support branch.

### CODE REVIEW

File: components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java [hunk 1]
```diff
@@ -133,6 +135,17 @@ public void addUserList(UserStoreManager userStore) throws UserAdminException {
         } catch (Exception e) {
             log.error(e.getMessage(), e);
```
Comment (hasinthaindrajee) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java]: This is wrong. e.getMessage is anyway inside e. So you should be more explicit while giving error message.

File: components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java [hunk 2]
```diff
@@ -133,6 +135,17 @@ public void addUserList(UserStoreManager userStore) throws UserAdminException {
...
             throw new UserAdminException(e.getMessage(), e);
```
Comment (hasinthaindrajee) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java]: Same as above comment

File: components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java [hunk 3]
```diff
@@ -133,6 +135,17 @@ public void addUserList(UserStoreManager userStore) throws UserAdminException {
...
+        } finally {
+            try {
+                if (csvReader != null) {
+                    csvReader.close();
+                }
+            } catch (IOException e) {
+                if (log.isDebugEnabled()) {
+                    log.debug(e);
+                }
+                log.error("Error occurred while closing CSV Reader");
```
Comment (hasinthaindrajee) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java]: Any reason why exception is only logged if debug enabled?. Please add the reason as a comment

Comment (ChamaraPhilipsuom) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java]: Fixed at #1682

File: components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java (hunk 2 above)
Comment (ChamaraPhilipsuom) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java]: Fixed at #1682

File: components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java (hunk 1 above)
Comment (ChamaraPhilipsuom) [inline components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java]: Fixed at #1682

**Pull request merged**